pkgs/development/libraries/alglib/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchurl, cmake, clang }:
+
+stdenv.mkDerivation rec {
+  pname = "alglib3";
+  version = "3.18.0";
+
+  src = fetchurl {
+    url = "http://www.alglib.net/translator/re/alglib-${version}.cpp.gpl.tgz";
+    sha256 = "sha256:0ag8dvcxzzp9riqvk4lhcbwhvh0lq54lbdnsbyr107rjfi2p1vlq";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    clang
+  ];
+
+  patches = [
+    ./patch-alglib-CMakeLists.patch
+  ];
+
+  meta = with lib; {
+    description = "ALGLIB is a cross-platform numerical analysis and data processing library.";
+    homepage = http://www.alglib.net/;
```

Comment:
Same as the other 2 reviews about these options.

Reply:
Done